modules/http/src/main/java/org/mule/module/http/internal/request/DefaultHttpRequesterConfig.java

```diff
@@ -39,6 +40,7 @@
 
 public class DefaultHttpRequesterConfig extends AbstractAnnotatedObject implements HttpRequesterConfig, Initialisable, Stoppable, Startable, MuleContextAware
 {
+    public static final String HTTP_CLIENT_FACTORY_NAME = "_httpClientFactory";
```

Comment:
Minor: Consider using same naming as used in MuleProperties. i.e. OBJECT_QUEUE_MANAGER = "_muleQueueManager";